test.js

```diff
@@ -320,13 +322,15 @@ test('is.inRange', t => {
 
 	t.true(m.inRange(x, 10));
 	t.true(m.inRange(0, 0));
+	t.true(m.inRange(-2, -3));
 	t.false(m.inRange(x, 2));
+	t.false(m.inRange(-3, -2));
 
-	t.throws(() => {
-		t.true(m.inRange(0));
-	});
+	t.throws(() =>
+		t.true(m.inRange(0))
+	);
 
-	t.throws(() => {
-		t.true(m.inRange(0, [5]));
-	});
+	t.throws(() =>
+		t.true(m.inRange(0, [5]))
+	);
```

Comment:
I don't want this change. It's intentionally a code-block. I only do inline arrow functions when the return value is actually used, for readability reasons.

Reply:
I do notice now that the `t.true` inside these `t.throws` statements are moot though. So you can remove those.

Reply:
I'll fix this if needed, but I'd like to ask first.
Why keep the `{ }`? IMO what adds to the readability is that the *throwing statement* is between new lines.

IMO there's nothing wrong with not wrapping a `void` statement in `{ }`, as they're eventually "dead characters", they don't have any meaning unless they contain more than two statements, that's just my opinion.

My example would be the `do` operator in `RxJS.Observable`. 
Often you do `.do(value => console.log(value))`. If I'd wrap every simple statement, I'd end up with something like: `.do(value => { console.log(value); })` which just clutters the code.

I'll revert it if you insist, but that's just my opinion.

Reply:
The following makes it perfectly clear `bar()` doesn't return anything and that `foo()` doesn't expect it to:

```js
foo(x => {
  bar();
});
```

This here, however, makes it seem like `bar()` returns something and `foo()` uses that return value:

```js
foo(x => bar());
```

I prefer explicit and clear code over saving a couple of characters.

Reply:
You know what, I do agree with you on this point. Will revert.